src/imgtools/cli/query.py

```
@@ -0,0 +1,114 @@
+import click
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)